ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Query/Version.hs

```diff
@@ -10,6 +10,9 @@ import           Ouroboros.Network.NodeToClient.Version
 data QueryVersion
   = TopLevelQueryDisabled
   -- ^ Only the @BlockQuery@ constructor of @Query@ is supported.
+  | TopLevelQueryV2
```

Comment:
I think this is right, but I would have expected `V1`, personally.

cc @DavidEichmann

Reply:
Agreed. Why not just `QueryV1`. The "Query" prefix matches `QueryVersion` and the `Query` names better.

I  unenthusiastically chose the longer `TopLevelQueryDisabled` name only because it's disabling **parts** of the `Query` type (all constructors other than `BlockQuery`) and `DisableQuery` sounded confusing if we were still using the `Query` type. Open to alternatives here.. maybe `NonBlockQuerysDisabled` :shrug:

Reply:
Ok, `QueryVersion1` then.